### PR TITLE
Update readme about the fork

### PR DESCRIPTION
**Pull request by bazsi**

### CODE REVIEW

**Comment by bazsi** [comment]
@fekete-robert can I get a review for this? thanks

File: README.md [hunk 1]
```diff
@@ -6,17 +6,48 @@
...
 [![Nightly](https://github.com/axoflow/axosyslog/actions/workflows/axosyslog-nightly.yml/badge.svg)](https://github.com/axoflow/axosyslog/actions/workflows/axosyslog-nightly.yml)
 [![Binary packages](https://github.com/axoflow/axosyslog/actions/workflows/packages.yml/badge.svg)](https://github.com/axoflow/axosyslog/actions/workflows/packages.yml)
 
-This repository contains the AxoSyslog source tree, cloud-ready syslog-ng images, and Helm charts
+This repository contains the AxoSyslog source tree, container images, and Helm charts
 created and maintained by [Axoflow](https://axoflow.com).
 
+## Quick-start
+
+In order to start using AxoSyslog, you need to choose one of these
```
Comment (fekete-robert) [inline README.md:22]:
```suggestion
To start using AxoSyslog, you can use one of these
```

**Comment by bazsi** [comment]
@fekete-robert I've merged your suggestions, can you add an "approve" to the PR?

thanks

**Review by fekete-robert — DISMISSED** [review]

File: README.md [hunk 2]
```diff
@@ -6,17 +6,60 @@
...
-This repository contains the AxoSyslog source tree, cloud-ready syslog-ng images, and Helm charts
+AxoSyslog started as a syslog-ng [[1]](#r1) fork, branched right after
+syslog-ng v4.7.1 with the following focus:
+  * cloud native (containers, helm charts, kubernetes integration),
+  * security data tailored parsing and transformation (filterx, app-parser, app-transform, etc)
+  * performance (eBPF, memory allocator, etc),
+
+AxoSyslog remains a drop in replacement for syslog-ng [[1]](#r1) uses the
+same license and development practices and was created by the original
+creators of syslog-ng [[1]](#r1).
```
Comment (fekete-robert) [inline README.md:25]:
```suggestion
AxoSyslog (created by the original creators of syslog-ng [[1]](#r1)):
- is a drop in replacement for syslog-ng [[1]](#r1),
- keeps using the same license and development practices.
```

**Review by fekete-robert — APPROVED** [review]

**Pull request merged**